src/libraries/System.Private.CoreLib/src/System/IO/StreamWriter.cs

```diff
@@ -942,8 +942,6 @@ private Task FlushAsyncInternal(bool flushStream, bool flushEncoder, Cancellatio
             }
 
             Task flushTask = Core(flushStream, flushEncoder, cancellationToken);
-
-            _charPos = 0;
             return flushTask;
```

Comment:
This can now be:
```C#
return Core(flushStream, flushEncoder, cancellationToken);
```
The separation was just to be able to have the charPos = 0 in the middle, but that's now gone.

Reply:
Indeed, should be ok now